modules/tasks/tasks/src/test/java/org/motechproject/tasks/web/TaskControllerTest.java

```diff
@@ -81,7 +89,7 @@
     public void setUp() throws Exception {
         initMocks(this);
         taskActionExecutor = new TaskActionExecutor(taskService, null, null);
-        triggerHandler = new TaskTriggerHandler(taskService, null, eventListenerRegistryService, null, taskActionExecutor, null);
+        triggerHandler = new TaskTriggerHandler(taskService, null, eventListenerRegistryService, null, taskActionExecutor, null, motechSchedulerService, schedulerTaskTriggerUtil);
```

Comment:
I think we are missing new tests for the task service (that test handling of scheduler tasks) and a test for the scheduler channel builder.